src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -289,6 +290,33 @@ def _validate_other_parameters(namespace, formula=None):
             namespace.os_type = formula.os_type
 
 
+def _validate_artifacts(namespace):
```

Comment:
You are a little late to the party 😁. I'm fine with this naming. It's called "validate" because it is used in the argument registration as a validator, so it is more natural when you type "validator=_validate_artifacts". It's pretty well known that validator is a terrible name since it is used for validation, transformation, etc. I would recommend sticking with the name.